src/cbmc/cbmc_parse_options.cpp

```diff
@@ -1188,6 +1201,7 @@ void cbmc_parse_optionst::help()
     " --yices                      use Yices\n"
     " --z3                         use Z3\n"
     " --refine                     use refinement procedure (experimental)\n"
+    " --pass                       use parameterized array for string solving (experimental)\n"
```

Comment:
Replaced it https://github.com/diffblue/cbmc/pull/341/commits/31b20a554f0c6e09917747a44e8a54e418e458f5